app/controllers/api/v1/categories_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::CategoriesController < ApplicationController
+
+  def index
+    render json: Category.all, adapter: :json
+  end
```

Comment:
Ideally the `adapter: json` should be set in your config. Check out the serializer reading in Rails week for guidance